source/includes/administration/_organizations.md

```diff
@@ -818,4 +818,124 @@ Returns an HTTP status code 200, with an empty response body.
 ```shell
 # Retrieve the organization's billing information
 curl -X DELETE "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e301bf5b86a/billing_info" \
-   -H "MC-Api-Key: your_api_key"
\ No newline at end of file
+   -H "MC-Api-Key: your_api_key"
+```
+
+
+<!-------------------- UPDATE BILLABLE ORGANIZATION INFORMATION -------------------->
+### Set billable organization information
+`PUT /organizations/:organization_id/billable`
+
+Set the organization to billable and update the billable organization information.
+
+```shell
+# Update billable organization info
+curl -X PUT "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e301bf5b86a/billable" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request_body"
+```
+> Request body example:
+
+```json
+{
+   "id": "39f6260d-f0ca-4be9-93c4-46405d471c04",
+   "billableStart": "2015-01-18",
```

Comment:
We need a table to know what can be updated. At the moment, only the billableEnd, billableStart and packageId can be changed.